src/applications/static-pages/mhv-signin-cta/components/messages/CustomAlert.jsx

```diff
@@ -16,7 +16,7 @@ import classNames from 'classnames';
  */
 const CustomAlert = ({
   children,
-  headerLevel = 2,
+  headerLevel = 3,
```

Comment:
The [`va-alert-sign-in` component defaults to `2`](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-alert-sign-in/va-alert-sign-in.tsx), fwiw